src/utils.jl

```diff
@@ -1,10 +1,10 @@
-# PooledString is an internal-only type for efficiently tracking string data + length
-# all strings indexed from a column/row will always be a full String
-# specifically, it allows avoiding materializing full Strings for pooled string columns while parsing
-# and allows a fastpath for materializing a full String when no escaping is needed
 export PooledString
 struct PooledString <: AbstractString end
 
+# PointerString is an internal-only type for efficiently tracking string data + length
+# all strings indexed from a column/row will always be a full String
+# specifically, it allows avoiding materializing full Strings for pooled string columns while parsing
```

Comment:
Is it the case that `PointerString` now indicates that strings should be pooled? Then what is `PooledString` for?